versioned/persist/nontx/src/main/java/org/projectnessie/versioned/persist/nontx/NonTransactionalDatabaseAdapterConfig.java

```diff
@@ -15,6 +15,32 @@
  */
 package org.projectnessie.versioned.persist.nontx;
 
+import org.immutables.value.Value;
 import org.projectnessie.versioned.persist.adapter.DatabaseAdapterConfig;
 
-public interface NonTransactionalDatabaseAdapterConfig extends DatabaseAdapterConfig {}
+public interface NonTransactionalDatabaseAdapterConfig extends DatabaseAdapterConfig {
+  int DEFAULT_REFERENCES_SEGMENT_SIZE = 250_000;
+  int DEFAULT_REF_LOG_STRIPES = 8;
```

Comment:
nit: Why decrease from the original 16? Does it hurt to have more stripes?

Reply:
Was a bit concerned about 16+1 stripes to read from, which "explodes" to (16+1)*20 = 340 initial reads at once.

Reply:
fair enough :+1: